flict/flictlib/format/markdown_format.py

```diff
@@ -10,71 +10,183 @@
+
+    def parsed_to_license(self, parsed_expr):
+        expr_type = parsed_expr['type']
+
+        if expr_type == 'license':
+            return parsed_expr['name']
+
+        if expr_type == 'operator':
+            name = parsed_expr['name']
+
+            operands = []
+            for op in parsed_expr['operands']:
+                if op['compatibility'] != "Yes":
+                    pass
+                else:
+                    parsed_op = self.parsed_to_license(op)
+                    operands.append(parsed_op)
+            operand_str = f" {name} "
+            expr = operand_str.join(operands)
+            if name == "OR" and len(operands) > 1:
+                expr = f" ( {expr} ) "
+            return expr
+
+    def packages_header(self):
+        return f"{self.headers['packages']} Packages\n"
+
+    def package_name_header(self, name):
+        return f"{self.headers['package_name']} {name}\n"
+
+    def dependencies_header(self):
+        return f"{self.headers['dependencies']} Dependencies\n"
+
+    def problem_header(self):
+        return f"{self.headers['problems']} Problems\n"
+
+    def dependency_header(self, name):
+        return f"{self.headers['dependency']} {name}\n"
+
+    def identified_license_header(self):
+        return f"{self.headers['identified_license']} Identified licenses in the combined work\n"
+
+    def output_compat_markdown(self, compats):
+        # print(str(compats))
+        result = []
+
+        result.append("# License compatibilities\n\n")
+
+        result.append("# Licenses\n\n")
+        for compat in compats['compatibilities']:
+            result.append(" * " + compat['license'] + "\n")
+
+        result.append("\n\n# Compatibilities\n\n")
+        for compat in compats['compatibilities']:
+            main_license = compat['license']
+            result.append(self._output_compat_markdown_licenses(main_license, compat))
+
+        return result
+
+    def format_verification(self, verification):
+        output = []
+        output.append(self.packages_header())
+        identified_license = None
+        for package in verification['packages']:
+
+            #            identified_license = package.get('identified_license', None)
+            #            identified_license_aliased = package.get('identified_license_aliased', identified_license)
+            identified_license = package.get('outbound_license', None)
+            identified_license_aliased = package.get('outbound_license_aliased', None)
+
+            identified_licenses = package.get('outbound_licenses', None)
+            #identified_licenses_aliased = package.get('outbound_licenses_aliased', identified_licenses)
+
+            output.append(self.package_name_header(package['name']))
+            description = package.get('description', "")
+            if description != "":
+                output.append(f"*Description*: {description}\n")
+
+            if identified_licenses == []:
```

Comment:
`if not any(identified_licenses)` - it's faster

Reply:
if not description:
   :)